api/app/schemas/hfi_calc.py

```diff
@@ -197,6 +198,20 @@ class HFIResultResponse(BaseModel):
     fire_start_ranges: List[FireStartRange]
 
 
+class HFIReadyState(BaseModel):
+    planning_area_id: int
+    hfi_request_id: int
+    ready: bool
+    create_timestamp: datetime
+    create_user: str
+    update_timestamp: datetime
```

Comment:
does the ready state need to distinguish between create and update? When are they used differently? (Or are you just following standard practice for DBs?)

Reply:
There will be a difference -- whoever makes the first change to a prep period will create the first ready record, then all other changes will be updates. Technically we mark `create_timestamp` and `update_timestamp` to be the same on creation, but yeah following standard practice there.